src/components/object-info/ObjectInfo.js

```diff
@@ -19,7 +19,8 @@ const objectInspectorTheme = {
 const nodeStyles = {
   'dag-cbor': { shortName: 'CBOR', name: 'CBOR', color: '#28CA9F' },
   'dag-pb': { shortName: 'PB', name: 'Protobuf', color: '#244e66' },
-  'git-raw': { shortName: 'GIT', name: 'Git', color: '#f14e32' },
+  'git-raw': { shortName: 'GIT', name: 'Git', color: '#378085' },
+  'raw': { shortName: 'RAW', name: 'Raw Block', color: '#f14e32' }, // eslint-disable-line quote-props
```

Comment:
the color for `git-raw` gets swapped for `raw` here. @lidel @rafaelramalho19

Reply:
This was intentional:  we used the same accent for git and `raw` blocks, and raw ones are way more important in CIDv1 world (used for leaves).  

I want to add `raw` and `dag-cbor` examples, but had no bandwidth for that, so for now only ensured we remove confusion between `git` and `raw`